components/BlurhashImage.vue

```diff
@@ -96,3 +68,15 @@ export default Vue.extend({
   }
 });
 </script>
+
+<style>
```

Comment:
Can this be scoped or does it break things?

Reply:
I'll check